CONTRIBUTING.md

```diff
@@ -416,7 +416,27 @@ Both images `packit-service` and `packit-worker` are built from source of curren
 
 As the last step playbook [zuul-tests.yaml](/files/zuul-tests.yaml) is executed.
 
+### Updating the test image
+
+The image in which tests are running
+(`quay.io/packit/packit-service-tests:stg`) is rebuilt every time a new commit
+shows up in the `main` branch. It is during this build when new dependencies
+can be installed or new versions of existing dependencies are pulled in.
+
+This is why adding a new test dependency (or modifying the Ansible playbooks
+configuring the test image in some other way) should be done in a PR
+_preceding_ the one in which this dependency is used the first time.
+
+In order to have `packit` updated to a version corresponding to the latest
```

Comment:
this should apply for `ogr` and `specfile` as well probably

Reply:
Added these, too and updated the wording accordingly.